compiler/core/js_dump.ml

```diff
@@ -524,6 +524,10 @@ and expression_desc cxt ~(level : int) f x : cxt =
          when Ext_list.length_equal el i
        ]}
     *)
+  | Call (e, el, {call_transformed_jsx = Some jsx_element}) ->
```

Comment:
Hope it ends up here and do something meaningful.
Unfortunately, it didn't work for me. It got lost somewhere.